kvstoremesh/.gitignore

```diff
@@ -0,0 +1 @@
+kvstoremesh
```

Comment:
Nit: It's a bit odd to me that the name of the executable is `kvstoremesh` maybe `kvstoremesh-apiserver` or `kvstore-agent`?

Reply:
Honestly I was doubtful about which name to pick, and I ended up using plain *kvstoremesh* mainly for brevity. I'm personally a bit against `kvstoremesh-apiserver` because it is not actually an apiserver (we already have the same problem with the clustermesh-apiserver, which makes understanding its role a bit difficult). The same applies for `kvstoremesh-operator`, because it is not strictly an operator in the kubernetes sense. I don't have strong opinion though.